### PR TITLE
Add examples

### PR DESCRIPTION
**Pull request by colleend**
add examples fix!

### CODE REVIEW

**Review by ievans — COMMENTED** [review]
@colleend can we add in Nathan's negative test case for the `yield` example?

**Review by nbrahms — APPROVED** [review]

File: python/lang/correctness/return-in-init.py [hunk 1]
```diff
@@ -36,3 +36,9 @@ def __init__():
     # OK, not in class
     yield 5
 
+class Ab:
+    def func1():
+        if not hello:
+            yield from blah
```
Comment (nbrahms) [inline python/lang/correctness/return-in-init.py:42]: I'm surprised this parses... `from` is a keyword that usually only works in imports.

Comment (nbrahms) [inline python/lang/correctness/return-in-init.py:42]:
```suggestion
            yield "blah"
```

File: python/lang/correctness/return-in-init.py [hunk 2]
```diff
@@ -36,3 +36,9 @@ def __init__():
...
+        yield from other
```
Comment (nbrahms) [inline python/lang/correctness/return-in-init.py:43]:
```suggestion
        yield other
```

**Pull request merged**